curriculum/challenges/english/07-scientific-computing-with-python/learn-list-comprehension-by-building-a-case-converter-program/657effaa2a5e0277d71f9cbe.md

```diff
@@ -7,13 +7,13 @@ dashedName: step-8
 
 # --description--
 
-By the end of the loop, `snake_cased_char_list` should contain all the converted characters in correct order. Use the `.join()` string method to convert the list of characters into a string.
+By this point, the `snake_cased_char_list` variable contains all the converted characters. You will need to convert the list of characters into a string using the `.join()` method.
 
 ```py
 ''.join(snake_cased_char_list)
 ```
```

Comment:
I would prefer a brief explanation of the `join` method instead of the example:
join is used on a string that will be the separator and takes as argument the list to merge, etc

Reply:
The `join` method has already been introduced, explaining it again would be repetition.